projects/ion/src/lib/tag/tag.component.html

```diff
@@ -0,0 +1,15 @@
+<div
+  data-testid="ion-tag"
+  [style.color]="!status && validateHexColor()"
```

Comment:
```suggestion
  [style.color]="!status && validateHexColor()"
```

This validation is repeated 3 times here, what do you think in create a single function to this?